### PR TITLE
Handle union in autocomplete

### PR DESCRIPTION
**Pull request by ymost**

### CODE REVIEW

File: test/test_table.py [hunk 1]
```diff
@@ -145,72 +168,208 @@ def test_column_retrieve_slow(self):
...
+        self.assertEqual(
+            [RecordedQuery('test_db', '.show database schema | project TableName, ColumnName, ColumnType | limit 10000')],
+            mock_kusto_client.recorded_queries,
+        )
         self.assertIsInstance(db.test_table_1.foo, StringColumn)
         self.assertIsInstance(db.test_table_1.bar, NumberColumn)
         self.assertIsInstance(db.test_table_2['baz'], BooleanColumn)
         self.assertIsInstance(db['other_table']['foo'], AnyTypeColumn)
 
+    def test_union_retrieve(self):
```
Comment (amos-rimon) [inline test/test_table.py]: these tests have a lot of code duplications, I think it can be reduced quite easily.

File: test/test_table.py [hunk 2]
```diff
@@ -7,7 +7,8 @@
 from pykusto.expressions import column_generator as col, StringColumn, NumberColumn, AnyTypeColumn, BooleanColumn
 from pykusto.query import Query
 from pykusto.type_utils import KustoType
-from test.test_base import TestBase, mock_columns_response, mock_tables_response, mock_databases_response, MockKustoClient
+from test.test_base import TestBase, mock_columns_response, mock_tables_response, mock_databases_response, \
+    MockKustoClient, RecordedQuery, mock_getschema_response
 
 
 class TestTable(TestBase):
```
Comment (amos-rimon) [inline test/test_table.py]: wouldn't it be better to split this into a few files? maybe test auto complete or test_db, test_column and test_client (I tend towards the latter)

File: pykusto/client.py [hunk 3]
```diff
@@ -207,42 +270,50 @@ def __getattr__(self, name: str) -> Any:
         """
         return self[name]
 
-    def get_table(self) -> KQL:
-        result = KQL(', '.join(self.tables))
-        if '*' in result or ',' in result:
-            result = KQL('union ' + result)
-        return result
+    @staticmethod
+    def static_is_union(*table_names: str) -> bool:
```
Comment (amos-rimon) [inline pykusto/client.py]: maybe move this to ItemFetcher?

File: test/test_table.py (hunk 2 above)
Comment (ymost) [reply test/test_table.py]: Split to `test_client` and `test_client_fetch`

File: test/test_table.py (hunk 1 above)
Comment (ymost) [reply test/test_table.py]: Consolidated tests to avoid duplication

File: pykusto/client.py (hunk 3 above)
Comment (ymost) [reply pykusto/client.py]: This method is Kusto-specific, while `ItemFetcher` is not

**Pull request merged**